code/modules/mob/living/carbon/xenomorph/facehuggers.dm

```diff
@@ -206,7 +213,7 @@
 		if(throwing || !lifetimer)//theres only a lifetimer as long as the huggers conscious
 			return
 	var/i = 10//So if we have a pile of dead bodies around, it doesn't scan everything, just ten iterations.
-	for(var/mob/living/carbon/M in view(4,src))
+	for(var/mob/living/M in view(4,src))
```

Comment:
why

Reply:
Hate arbitrarily limiting things to carbon, but in this case I suppose it's fine as is for performative reasons in light of the lack of non-carbons for which this would matter.

Reply:
Do so then please